genshin/models/hoyolab/record.py

```diff
@@ -110,8 +110,8 @@ class HoyolabUserCertification(APIModel):
     For example artist's type is 2.
     """
 
-    icon_url: str
-    description: str = Aliased("desc")
+    icon_url: typing.Optional[str] = None
+    description: typing.Optional[str] = None
```

Comment:
Oops sorry one last thing, this is missing it's alias. `= Alias("desc", default=None)` should work I believe.

Reply:
Lol, It is Aliased. Should be fixed.